### DISCUSSION TITLE
Should we enable `log-to-file` by default for the `server` binary?

### DISCUSSION THREAD

**Discussion by hasezoey** [tramhao/termusic#262] [Ideas]
Should we maybe enable `log-to-file` by default for the `server` binary? as a alternative to that we could also only enable it if started from the `tui` binary.
Maybe even enable for the tui binary?

if we enable it for the server binary (especially when started via the tui binary), then it will be more likely bug reports will at least have the errors in the logs without having to re-run the app and try to reproduce it (this should catch some random errors which might not be easily reproduce-able) and should also clarify (at least a bit) to the users why the server may have crashed (instead of being completely silent when started via the tui binary).

Note: the default log level is `warn` (if i recall correctly)

**Comment by tramhao** [comment]
yes that is great . Especially for tui. as it will clear screen and have to log into file to see errors.

**Reply by hasezoey** [reply]
for reference, pr available at #264

**Comment by hasezoey** [comment]
Since #264 (released in `0.9.1`) `--log-to-file` is enabled by default (for both server & tui)
